org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/discovery/ZigBeeDiscoveryService.java

```diff
@@ -86,7 +87,9 @@ protected void addZigBeeCoordinatorHandler(ZigBeeCoordinatorHandler coordinatorH
         ZigBeeNetworkNodeListener listener = new ZigBeeNetworkNodeListener() {
             @Override
             public void nodeAdded(ZigBeeNode node) {
-                ZigBeeDiscoveryService.this.nodeDiscovered(coordinatorHandler, node);
+                if (searchStarted) {
```

Comment:
Maybe `scanStarted` is a better name for the variable as it better describes its purpose. WDYT?
I'm wondering if `searchStarted` needs to be synchronized as it can be accessed from different threads.

Reply:
I have changed the name to `scanStarted`. It makes more sense, since the rest of the code is also talking about a "scan", rather than a "search". Thanks for pointing this out.